Polished Node.js version documentation (#1572)
zeit/docs: components/references-mdx/runtimes/official-runtimes/official-runtimes.mdx

### Node.js Version

> When creating a new deployment, it automatically receives the latest LTS Node.js version available on ZEIT Now at that point in time.

Reviewer: ```suggestion
New deployments automatically select the latest LTS Node.js version available on ZEIT Now at that point in time.
```

> In order to customize it, add a `package.json` file with an `engines` field, like so:

Reviewer: ```suggestion
In order to select a different version, add a `package.json` file with an `engines` field, like so:
```

  <InlineCode>package.json</InlineCode> file.
</Caption>

> When defining an `engines` property, the following Node.js versions can be supplied:

Reviewer: ```suggestion
When defining an `engines` property, the following Node.js versions can be selected:
```

- **10.x**
- **12.x** (The default if the property is not defined)

> In order to find out which Node.js version your deployment is using, run `node -v` inside it or log the output of `process.version`.

Reviewer: ```suggestion
In order to find out which Node.js version your deployment is using, run `node -v` inside the `build` script or log the output of `process.version`.
```

Author: The `build` script is a `package.json` concept. We can't rely on that.

<Note type="warning">
  Our upstream providers do not support passing a specific version like{' '}

>   <InlineCode>12.14.0</InlineCode>. Only "more loose" values like{' '}

Reviewer: ```suggestion
  <InlineCode>12.14.0</InlineCode>. Only major values like{' '}
```
